src/cli/CHANGELOG.md

```diff
@@ -1,10 +1,13 @@
-﻿# Changelog
+# Changelog
 All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 
 Prefix your items with `(Template)` if the change is about the template and not the resulting application.
 
+## 0.20.0
+- Configured MobSF security scan on Android and iOS for Staging and Production builds. 
```

Comment:
This is a significant change and you must update the documentation (under the doc folder, and perhaps even mention this feature in the readme).

Reply:
docs updated